phpunit/class-block-library-comment-template-test.php

```diff
@@ -394,7 +394,7 @@ function test_rendering_comment_template_unmoderated_preview() {
 		// Here we use the function prefixed with 'gutenberg_*' because it's added
 		// in the build step.
 		$this->assertEquals(
-			'<ol class="wp-block-comment-template"><li id="comment-' . self::$comment_ids[0] . '" class="comment even thread-odd thread-alt depth-1"><div class="wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/author-url/" target="_self" >Test</a></div><div class="wp-block-comment-content"><p>Hello world</p></div></li><li id="comment-' . $unapproved_comment[0] . '" class="comment odd alt thread-even depth-1"><div class="wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/unapproved/" target="_self" >Visitor</a></div><div class="wp-block-comment-content"><p><em class="comment-awaiting-moderation">Your comment is awaiting moderation.</em></p>Hi there! My comment needs moderation.</div></li></ol>',
+			'<ol class="wp-block-comment-template"><li id="comment-' . self::$comment_ids[0] . '" class="comment even thread-odd thread-alt depth-1"><div class="wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/author-url/" target="_self" >Test</a></div><div class="wp-block-comment-content"><p>Hello world</p></div></li><li id="comment-' . $unapproved_comment[0] . '" class="comment odd alt thread-even depth-1"><div class="wp-block-comment-author-name">Visitor</div><div class="wp-block-comment-content"><p><em class="comment-awaiting-moderation">Your comment is awaiting moderation.</em></p>Hi there! My comment needs moderation.</div></li></ol>',
```

Comment:
This will need a further rebase following 806a742fcad8286ff6f9dcc9de7384bd376e8709 -- the comment text will now have paragraph tags of its own.

Reply:
Sorry @peterwilsoncc, I'm not quite sure what you mean here. 😅  That commit (https://github.com/WordPress/gutenberg/commit/806a742fcad8286ff6f9dcc9de7384bd376e8709) is already in `trunk`, and this branch doesn't have any merging conflict. 👀 

Do you mean adding a `<p></p>` tag to the content of that comment so this test covers that case also?